src/conf_mode/service_ipoe-server.py

```diff
@@ -29,9 +30,92 @@
 from vyos import airbag
 airbag.enable()
 
+
 ipoe_conf = '/run/accel-pppd/ipoe.conf'
```

Comment:
I wonder if this directory path is common enough to deserve a constant in a reusable module.